README.md

```diff
@@ -507,9 +336,15 @@ If you'd like to contribute to this project, you'll first need to clone this rep
 Then, you will need to run the following commands:
 
 1. `poetry install` -- installs the necessary dependencies into a poetry-managed virtual environment.
-2. `poetry run pre-commit install` -- creates a git [pre-commit](https://pre-commit.com) hook which will automatically run formatters, style checks, etc. against your proposed commits.
+2. ~`poetry run pre-commit install` -- creates a git [pre-commit](https://pre-commit.com) hook which will automatically run formatters, style checks, etc. against your proposed commits.~
+
+To run commands from source as you edit, use the `poetry run` command. For example:
+
+```
+poetry run aerie-cli plans simulate --id 42
+```
 
-### Deployment
+<!-- ### Deployment
```

Comment:
Commented out section

Reply:
Commented out but I hope to deploy to PyPI in the future and bring this section back. I can completely delete it for now if that's better.

Reply:
Pulled it out and noted it in an issue for future reference.